docs/source/frameworks.rst

```diff
@@ -115,6 +115,17 @@ Example Projects:
 .. autoclass:: bentoml.frameworks.lightgbm.LightGBMModelArtifact
 
 
+========
+CatBoost
+========
+
+Example Projects:
+
+* Titanic Survival Prediction -  `Google Colab <https://colab.research.google.com/github/bentoml/gallery/blob/master/lightbgm/titanic-survival-prediction/lightbgm-titanic-survival-prediction.ipynb>`__ / `Notebook Source <https://github.com/bentoml/gallery/blob/master/lightbgm/titanic-survival-prediction/lightbgm-titanic-survival-prediction.ipynb>`__
```

Comment:
Do you plan to add an example notebook for CatBoost in the `bentoml/gallery`  repo?